site/content/posts/contributor-of-the-month.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Introducing the Contributor of the Month initiative"
+slug: contributor-of-the-month
+date: 2022-03-01
+author: Cartographer Team
+authorLocation: https://github.com/vmware-tanzu/cartographer/blob/main/MAINTAINERS.md
+image: /img/posts/contributor-of-the-month/Contributor-of-the-month-v3.png
+excerpt: "A monthly program to reward contributions"
+tags: ['community']
+---
+
+Since October 2021, when the Cartographer project was made public, we have been very fortunate to count with individuals who have helped improve the codebase and docs while also telling the Cartographer story in conferences, live streams and other forms of content. All of them have done this out of intrinsic motivation, without expecting any reward or public recognition and we are forever grateful to them for this.
+
+Nevertheless, we would like to fill in the [extrinsic motivation](https://www.rochester.edu/emerging-leaders/understanding-intrinsic-and-extrinsic-motivation/#:~:text=Intrinsic%20motivation%20involves%20performing%20a,punishment%20or%20receiving%20a%20reward) void by rolling out a small initiative called **Contributor of the Month**
+
+**How does it work?**
+
+We plan to recognize contributors in three different categories:
+
+* Code contributor (merged PRs)
+* Content creator (tweets/blog posts/screencasts/presentations/podcast)
+* Project and community supporter (community meetings participation/Slack interactions)
+
+Code contributions will be tracked using the `contributor shoutouter` program located in [this repo](https://github.com/vmware-tanzu/community-engagement/tree/main/shoutouter) while content and community support will be tracked down by the Cartographer community team.
+
+As a little token of appreciation, contributors will receive branded Cartographer swag that we hope you find useful and conscious.
```

Comment:
```suggestion
As a little token of appreciation, contributors will receive Cartographer branded, sustainable swag that we hope you find useful.
```
Just not sure folks would know what you mean by 'conscious'?

Reply:
Yeah, it's more 'environmentally responsible', or even better, 'sustainable'. I'm not sure how t-shirts would fall into that category but stickers not, but will keep refining swag. The goal here was to keep it generic so we can iterate on swag without having to change the post